exec/operator.go

```diff
@@ -4,7 +4,9 @@
 
 package exec
 
-import "robpike.io/ivy/value"
+import (
+	"robpike.io/ivy/value"
+)
```

Comment:
restore layout

Reply:
Fixed. I must have added a print and then when I deleted it and goimports removed the "fmt" import, it left the block. I gave up on trying to convince the goimports owners to change this behavior many years ago and just defer to the tool now. But fixed.